vespa/deployment.py

```diff
@@ -874,8 +1092,44 @@ def get_dev_region(self) -> str:
         return self._request("GET", "/zone/v1/environment/dev/default")["name"]
 
     def get_prod_region(self):
-        # TODO Support multiple regions
-        return self.application_package.deployment_config.regions[0]
+        regions = self.get_prod_regions()
+        return regions[0]
```

Comment:
Any plans to support arbitrary prod regions in `get_application()`?

Reply:
Added support for that now.